pkg/frontend/options/options.go

```diff
@@ -34,6 +36,14 @@ type Options struct {
 	TLSListenPort int `yaml:"tls_listen_port,omitempty"`
 	// ConnectionsLimit indicates how many concurrent front end connections trickster will handle at any time
 	ConnectionsLimit int `yaml:"connections_limit,omitempty"`
+	// MaxRequestBodySize indicates the maximum allowed size of the request body.
+	// If the body is too large. Trickster will truncate the payload or return a
+	// 413 Payload Too Large response depending upon truncate_request_body_too_big.
+	// Use 0 for no body allowed, and < 0 for no maximum.
+	MaxRequestBodySizeBytes *int64 `yaml:"max_request_body_size_bytes"`
```

Comment:
q: why is this a pointer?

we're applying a default (L64), so it doesn't seem like this is to allow us to tell if the field is unset, unless i'm missing something subtle.

Reply:
This matters.  The default is 10MB when the pointer is nil, but the user can supply `0` to indicate no body allowed. So if this is not a pointer, it is not possible to know if the user wants to use the default (10MB) or 0.